src/PeoplePage.1.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { Person } from './types';
+import { PersonLink } from './PersonLink';
+import { Loader } from './components/Loader';
+import { getPeople } from './api';
+import { TableHeaders } from './types/TableHeaders';
+
+export const PeoplePage: React.FC = () => {
```

Comment:
Why the name of the file is `PeoplePage.1.tsx`?

Reply:
this is VS code autocorrection, when I wanted to transfer the logic into another file, VS code corrected the name itself and I didn't notice it